src/test/regress/expected/multi_partitioning.out

```diff
@@ -1028,6 +1028,29 @@ CREATE TABLE partitioned_users_table_2009 PARTITION OF partitioned_users_table F
 CREATE TABLE partitioned_events_table_2009 PARTITION OF partitioned_events_table FOR VALUES FROM ('2017-01-01') TO ('2018-01-01');
 INSERT INTO partitioned_events_table SELECT * FROM events_table;
 INSERT INTO partitioned_users_table_2009 SELECT * FROM users_table;
+-- test distributed partitions are indeed colocated with the parent table
+CREATE TABLE sensors(measureid integer, eventdatetime date, measure_data jsonb, PRIMARY KEY (measureid, eventdatetime, measure_data))
+PARTITION BY RANGE(eventdatetime);
+CREATE TABLE sensors_old PARTITION OF sensors FOR VALUES FROM ('2000-01-01') TO ('2020-01-01');
+CREATE TABLE sensors_2020_01_01 PARTITION OF sensors FOR VALUES FROM ('2020-01-01') TO ('2020-02-01');
+CREATE TABLE sensors_new PARTITION OF sensors DEFAULT;
+SELECT create_distributed_table('sensors', 'measureid', colocate_with:='none');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT logicalrelid, colocationid FROM pg_dist_partition
```

Comment:
Minor: maybe do not print the `colocationId` as that might change if other tests change the colocationId.
```SQL
SELECT 
    count(DISTINCT colocationid)
FROM 
    pg_dist_partition
WHERE logicalrelid IN ('sensors'::regclass, 
                                        'sensors_old'::regclass, 
                                        'sensors_2020_01_01'::regclass, 
'sensors_new'::regclass);
```